patterns-base/src/main/scala/codacy/patterns/Custom_Scala_FieldNamesChecker.scala

```diff
@@ -13,10 +13,10 @@ class Custom_Scala_FieldNamesChecker(configuration: Custom_Scala_FieldNamesCheck
     tree.collect{
       //val definitions
       case t@q"..$_ val ..$patsnel: $tpe = $expr" if configuration.includeEnums || ! isEnumValDefRegexOrDecl(t.parent,Option(expr),tpe) =>
-        conflictingNames(patsnel)
+        conflictingNames(patsnel, isConstant(t))
       //val declarations
       case t@q"..$_ val ..$pnamesnel: $tpe" if configuration.includeEnums || ! isEnumValDefRegexOrDecl(t.parent,tpe=Option(tpe)) =>
-        conflictingNames(pnamesnel)
+        conflictingNames(pnamesnel, isConstant(t))
```

Comment:
i think you might not need this check here. it's a declaration (has no rhs and therefore no literal) in an object this can't really happen.